reflex/utils/prerequisites.py

```diff
@@ -29,6 +29,26 @@
 from reflex.utils import console, path_ops, processes
 
 
+def check_latest_package_version(package_name: str):
+    """Check if the latest version of the package is installed.
+
+    Args:
+        package_name: The name of the package.
+    """
+    url = f"https://pypi.org/pypi/{package_name}/json"
+
+    response = httpx.get(url)
```

Comment:
We may want to put this in a try/except so that they can still run their app if not connected to the internet